gradle.properties

```diff
@@ -1 +1,27 @@
-kotlinVersion=1.3.50
\ No newline at end of file
+# IntelliJ Platform Artifacts Repositories
+# -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
+
+pluginGroup = com.github.shiraji
+pluginName = Plugin Importer+Exporter
+pluginVersion = 0.0.5
+
+# See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
+# for insight into build numbers and IntelliJ Platform versions.
+pluginSinceBuild = 211
```

Comment:
No. I don't see why I have to update to 211 as a minimum version of this plugin.